chromium_src/content/public/common/content_features.cc

```diff
@@ -19,6 +19,7 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kWebNfc, base::FEATURE_DISABLED_BY_DEFAULT},
 #endif
     {kWebOTP, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kServiceWorkerTerminationOnNoControllee, base::FEATURE_ENABLED_BY_DEFAULT},
```

Comment:
sort please

Reply:
Also please add this feature in `BraveMainDelegateBrowserTest.EnabledFeatures` test